support-frontend/assets/helpers/abTests/abtest.js

```diff
@@ -192,10 +231,22 @@ const trackOptimizeExperiment = (optimizeId: string, variants: Variant[], varian
 function assignUserToVariant(
   mvtId: number,
   test: Test,
+  acquisitionDataTest: ?AcquisitionABTest,
 ): number {
-  const { independent, seed } = test;
+  const { referrerControlled, seed } = test;
 
-  return randomNumber(mvtId, independent, seed) % test.variants.length;
+  if (referrerControlled && acquisitionDataTest != null) {
+    const acquisitionVariant = acquisitionDataTest.variant;
+    const index = test.variants.findIndex(variant => variant.id === acquisitionVariant);
+    if (!index) {
+      console.error('Variant not found for A/B test in acquistion data');
```

Comment:
typo in 'acquisition'

Reply:
There's the need for that keyboard shortcut! 😄